libretro-common/include/libretro.h

```diff
@@ -2569,6 +2569,31 @@ enum retro_mod
  */
 #define RETRO_ENVIRONMENT_GET_FILE_BROWSER_START_DIRECTORY 80
 
+/**
+ * Sets a pointer to arbitrary data for the actively running core.
```

Comment:
I would add a note here about the use case of this environment call. Something like: `Intended for use as a substitute for global state, which is a common source of bugs.`

Reply:
Added in the description below :+1: